### ISSUE TITLE
Move column threshold option

### ISSUE THREAD

**Issue by iorzetic** [angular-ui/ng-grid#3507]
Could we add a moving column threshold option so that column moving does not start until the threshold is met?

Short description of our case: Because the column menu button is quite small for using with a tablet or mobile device, we decided that we want to add a custom header template that makes the whole header act as a button for the menu. This causes problems while trying to access the menu because not every mouse click is perfectly still, sometimes the user triggers a move action on that. Therefore we thought that adding a threshold option for when moving a column should start could fix this.

Are there any alternative solutions for this?

**Comment by PaulL1** [comment]
Yeah, there are a bunch of move things to look at, this one makes sense too.

**Comment by AgDude** [comment]
I think this would solve #3333 also
